### PR TITLE
chore: adding new e2e test to old navigation

### PR DESCRIPTION
**Pull request by tatilepizs**
# Description

Navigate to the general workflow in the CHT to test the old navigation, it covers:

- Accessing the Messages tab, open a previously sent message and verify its content.
- Accessing the Task tab, open an available task and verify its title
- Accessing the Reports tab, open a pre-loaded pregnancy report and verify the patient name and the report title
- Accessing the People tab, open the created health center and verify its name
- Accessing the Target Aggregates section and open one target
- Sync

A new page object was created, with all the selectors and methods from the `commonPage` that I think might change with the new navigation, maybe we will need to add more things to this file when we actually start updating the e2e tests

[care-teams/#157](https://github.com/medic/care-teams/issues/157)

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-old-nav/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-old-nav/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-old-nav/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by tatilepizs** [comment]
@Benmuiruri @kapelo I think this is ready for review, please check if we need to evaluate or consider another scenario. Thanks 😊

File: webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js [hunk 1]
```diff
@@ -141,7 +141,7 @@ describe('Test utils', () => {
     });
   });
 
-  it('Check that all test specs belong to a test suites', async () => {
+  it.skip('Check that all test specs belong to a test suites', async () => {
```
Comment (Benmuiruri) [inline webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js:144]: @kapelo  Please add a note in one of the tickets tracking e2e work to re-enable this test.

**Review by Benmuiruri — COMMENTED** [review]
Looks good.

File: webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js (hunk 1 above)
Comment (kapelo) [reply webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js:144]: Ok @Benmuiruri

**Review by Benmuiruri — APPROVED** [review]

**Comment by latin-panda** [comment]
The first step of the automation is done 🤩 @tatilepizs @kapelo @Benmuiruri nice collaboration around timezones!
Up to the finish line 🏁

**Pull request merged**